virttest/utils_hotplug.py

```diff
@@ -0,0 +1,388 @@
+#
+# library for hotplug(cpu) related helper functions
+# can be extended to memroy related helper functions aswell
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; specifically version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: IBM (c) 2017
+# Author: Satheesh Rajendran <sathnaga@linux.vnet.ibm.com>
+
+
+import re
+import json
+import logging
+import platform
+
+import virsh
+import libvirt_xml
+import utils_misc
+import utils_test
```

Comment:
let us have import statement as `from . import virsh` and same for other to have a explicit difference of python packages and Avocado packages